src/registry/children/ChildrenTable.tsx

```diff
@@ -0,0 +1,193 @@
+}
+
+const ChildDeleteButton: React.FC<ChildDeleteButtonProps> = ({child, onRemoveChild}) => {
+  const intl = useIntl();
+  return (
+    <SubtleButton
+      onClick={onRemoveChild}
+      hint="danger"
+      aria-label={intl.formatMessage(
+        {
+          description: "Children component: children table 'delete child' accessible label",
+          defaultMessage: `{hasFirstNames, select,
+            true {Delete child with first name: {firstNames}}
+            other {Delete child with BSN: {bsn}}}`,
+        },
+        {
+          hasFirstNames: !!child.firstNames,
+          firstNames: child.firstNames,
+          bsn: child.bsn,
+        }
+      )}
+    >
+      <UtrechtIcon>
+        <Icon icon="remove" />
+      </UtrechtIcon>
+    </SubtleButton>
+  );
+};
+
+const EmptyMessage: React.FC = () => (
+  <i>
+    <FormattedMessage description="Empty field message" defaultMessage="No information provided" />
+  </i>
+);
+
+export interface ChildrenTableProps {
+  name: string;
+  values: ExtendedChildDetails[];
+  enableSelection?: boolean;
+  onSelectionChanged: () => void;
+  updateChild: (childIndex: number, child: ExtendedChildDetails) => void;
+  removeChild: (childIndex: number) => void;
+}
+
+const ChildrenTable: React.FC<ChildrenTableProps> = ({
+  name,
+  values,
+  enableSelection,
+  onSelectionChanged,
+  updateChild,
+  removeChild,
+}) => {
+  const intl = useIntl();
+  return (
+    <Table>
+      <TableHeader>
+        <TableRow>
+          {enableSelection && <TableHeaderCell scope="col" />}
+          <TableHeaderCell scope="col">
+            <FormattedMessage
+              description="Children component: children table 'bsn' header"
+              defaultMessage="BSN"
+            />
+          </TableHeaderCell>
+          <TableHeaderCell scope="col">
+            <FormattedMessage
+              description="Children component: children table 'firstNames' header"
+              defaultMessage="First names"
+            />
+          </TableHeaderCell>
+          <TableHeaderCell scope="col">
+            <FormattedMessage
+              description="Children component: children table 'dateOfBirth' header"
+              defaultMessage="Date of birth"
+            />
+          </TableHeaderCell>
+        </TableRow>
+      </TableHeader>
+      <TableBody>
+        {values.map((child, index) => (
+          <TableRow key={child.__id || index}>
+            {enableSelection && (
+              <TableCell>
+                <BareCheckbox
+                  name={`${name}.${index}.selected`}
+                  aria-label={intl.formatMessage(
+                    {
+                      description: "Children component: child 'selected' aria label",
+                      defaultMessage: 'Include {firstname}?',
+                    },
+                    {
+                      firstname: child.firstNames,
+                    }
+                  )}
+                  onChange={onSelectionChanged}
```

Comment:
shouldn't we use `onBlur` instead of `onChange` for consistency? Every other field is marked as touched when the user moves focus away, not when the value changed.

The idea is: the user has interacted with the field, so it is now touched, so now we can display any validation errors that may exist instead of overwhelming them with validation errors before they've even touched the field.